students/aravichander/lesson06/Mailroom3.py

```diff
@@ -0,0 +1,89 @@
+#Mailroom #2
+
+import sys
+import json
+import unittest 
+
+Donors = {"Bill Gates":[10000,12000],
+		  "Jeff Bezos":[50],`
+		  "Mark Zuckerberg": [500,600,700],
+		  "Paul Allen": [250,350,450],
+		  "King of Siam": [200,250]
+		  }
+
+def printdonors():
+	for x,v in Donors.items():
+		print(x,"donated $",v)
+
+def thankall():
+	for x in Donors:
```

Comment:
you are working with botht he key and the value, so better to use:

```
for donor, donations in Donors.items():
    letterofthanks(donor, sum(donations))
```

and avoid using 'x' when there is a more meaningful name available.